internal/mtail/mtail.go

```diff
@@ -104,6 +106,33 @@ func (m *Server) StartTailing() error {
 	return nil
 }
 
+// 定时扫描通配符目录地址，对通配符地址扫描追加
+func (m *Server) TimingTailing() error {
+	// 判断地址中是否含有通配符
+	hasMeta := func(path string) bool {
```

Comment:
There's a second hasMeta you've added in tail.go, would you make that an Exported method and then reuse it from there?